docs/modules/datadog_aws_integration_v2.md

```diff
@@ -0,0 +1,34 @@
+# datadog_aws_integration_v2
+
+**datadog_aws_integration_v2** Creates the required IAM roles and polices for Datadog to integrate with your AWS account. It will also create the aws integration in datadog.
+
+## Example Usage
+
+```terraform
+module "datadog_aws_integration_v2" {
+  source = "git@github.com:RSS-Engineering/terraform//modules/datadog_aws_integration_v2?ref={commit}"
+
+  app_key = "" # Recommended to use the kms_secrets module here
+  api_key = "" # Recommended to use the kms_secrets module here
+
+  tags = [
+    "account:${data.aws_caller_identity.current.account_id}",
+    "region:${data.aws_region.current.name}",
+    "service:Application",
+    "env:production"
+  ]
+
+  included_metrics = ["AWS/SQS", "AWS/ElasticMapReduce"]
+}
+```
+
+## Argument Reference
+
+---
+
+The following arguments are supported:
+
+- `app_key` - (required) The Datadog application key.
+- `api_key` - (required) The Datadog API key.
+- `tags` - (optional) A list of tags to to apply to all metrics in the account.
+- `included_metrics` - (optional) List of metric namespaces to be collected. Use [datadog_integration_aws_available_namespaces](https://registry.terraform.io/providers/DataDog/datadog/latest/docs/data-sources/integration_aws_available_namespaces) data source to get allowed values or you can find them at [AWS services metrics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/aws-services-cloudwatch-metrics.html). Nothing will be collected if this is not set.
```

Comment:
Would most consumers want all the metrics by default?  It would make it easier to add this integration without needing to list all the metrics.  The default metric filters use the [exclude_only](https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/integration_aws_account#exclude_only-1) field.  What if we expose that here instead, and have it default to the same exclusions that the provider uses by default?